### ISSUE TITLE
ToggleSwitch internally sets MinWidth to 154, which is way larger than its default Width

### ISSUE THREAD

**Issue by eriklimakc** [unoplatform/uno#13946]
### Current behavior

Ref. to https://github.com/microsoft/microsoft-ui-xaml/issues/3652

![image](https://github.com/unoplatform/uno/assets/114886335/07f16719-d7b7-49fc-888e-d3fe7ab5aba1)

```
<StackPanel>
	<TextBlock Text="Fluent" />
	<StackPanel Orientation="Vertical" Spacing="10" Margin="10">
		<ToggleSwitch IsOn="True" MinWidth="53" Background="Red" />
		<ToggleSwitch IsOn="True" Background="Red" />
		<ToggleSwitch IsOn="True" Background="Red" MinWidth="154" />
	</StackPanel>

	<TextBlock Text="Material" />
	<StackPanel Orientation="Vertical" Spacing="10" Margin="10">
		<StackPanel.Resources>
			<MaterialTheme xmlns="using:Uno.Material" />
		</StackPanel.Resources>
		<ToggleSwitch IsOn="True" MinWidth="53" Background="Red" />
		<ToggleSwitch IsOn="True" Background="Red" />
		<ToggleSwitch IsOn="True" Background="Red" MinWidth="154" />
	</StackPanel>
</StackPanel>
```

The `MinWidth="154"` is defined in:

 - [Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml#L3972](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml#L3972)

 - [Uno.UI.FluentTheme.v1/themeresources_v1.xaml#L19731](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml#L19731)

 - [Uno.UI.FluentTheme.v2/themeresources_v2.xaml#L23315](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml#L23315)
 
Repro in Playground: http://playground.platform.uno/#ccfdf8c8

### Workaround

Use this setter to the style `<Setter Property="MinWidth" Value="0" />`.

### Works on UWP/WinUI

No

### Affected platforms

WebAssembly, Android, iOS, Skia (WPF), Skia (GTK on Linux/macOS/Windows), Skia (Linux Framebuffer), Windows

**Comment by agneszitte** [comment]
I think https://github.com/unoplatform/uno/issues/13946 is a Microsoft issue (unrelated to uno) but we will need to double-check to be sure.
1- Make sure our current default style in Uno matches the latest version of Microsoft
2- Testing the control in a Microsoft app outside of Uno to see the current behavior

**Comment by Youssef1313** [comment]
@agneszitte Yes it's a Microsoft issue. There is an open issue: https://github.com/microsoft/microsoft-ui-xaml/issues/3652

**Comment by agneszitte** [comment]
I added this https://github.com/microsoft/microsoft-ui-xaml/issues/3652 issue to the tracking issue that we have https://github.com/unoplatform/uno/issues/982

**Comment by ForkBug** [comment]
Thank you @erikvilima .This helped me to find out why the native ios switch control is too wide.

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not actionable as a fresh runtime-test repro in this repo.

**Reason:** The defect originates in WinUI's default `ToggleSwitch` style (`MinWidth=154`) which Uno intentionally mirrors via `SystemResources.xaml` and the v1/v2 Fluent theme resources. The issue is already tracked as a WinUI parity item (`waiting-for-winui` label) referencing [microsoft/microsoft-ui-xaml#3652](https://github.com/microsoft/microsoft-ui-xaml/issues/3652). Verifying the symptom is trivial via the linked playground (http://playground.platform.uno/#ccfdf8c8); the fix in Uno depends on the upstream WinUI decision rather than a local code change.
